MarsOnboardingTask/StepDefinitions/LanguagesStepDefinitions.cs

```diff
@@ -0,0 +1,112 @@
+using MarsOnboardingTask.Pages;
+using MarsOnboardingTask.Utils;
+using NUnit.Framework;
+
+
+namespace MarsOnboardingTask.StepDefinitions
+{
+    [Binding]
+    public class LanguagesStepDefinitions : Hooks
+    {
+        Login loginPageObj = new Login();
+        Languages languageObj = new Languages();
+
+        [Given(@"User logs into Mars application")]
+        public void GivenUserLogsIntoMarsApplication()
+        {
+            loginPageObj.LoginInToApplication(driver);
+        }
+
+        [When(@"User adds '([^']*)' language with '([^']*)' language level into the languages list")]
+        public void WhenUserAddsLanguageWithLanguageLevelIntoTheLanguagesList(string language, string languageLevel)
+        {
+            languageObj.AddLanguageWithlanguageLevel(driver, language, languageLevel);
```

Comment:
do not pass the driver in action methods, Instead make use of Common driver by extending the class or pass the driver when creating the page object via constructor

Reply:
updated as per the review comments